integr8ly/integreatly-operator
# MGDAPI-3552 bump version of CRO in RHOAM to v0.37.1

# Issue link

MGDAPI-3552

# What

bump CRO version , contains postgres version bump 10.18


# Verification steps



- install RHOAM on CCS with useclusterstorage set to 'false'
- check the Postgres CR's status for the version it should be `10.18`
- check the operatorhub installed operator CRO version should be v0.37.1
- check the rhmi CR the version of CRO should be v0.37.1
